### ISSUE TITLE
What's configure when you trained LiTS dataset ?

### ISSUE THREAD

**Issue by TakumiItoPappare** [15029257158/QAU-Net#1]
I trained the Qau-Net with the LiTS dataset, but the dice accuracy of its model is far lower than your best 85.90%.

The code your team uploaded to Github lacks some important points, so I looked at the paper and tried to reproduce the original code.
Is there anything wrong with the following?
　
　
　
There was no code for normalization on Github, so I wrote the following while looking at the paper.
Is it right?
Preprocessing LiTS dataset:
・ct_array is dicm data from LiTS dataset.
```
ct_array[ct_array > 250] = 250
ct_array[ct_array < -200] = -200
ct_array = ct_array.astype(np.float32)
ct_array = ((ct_array - (-200) * 2) / (250 - (-200))) - 1
```

・Segmentation labels in the LiTS dataset have [0, 1, 2], so out_ch of QAU_Net(in_ch, out_ch) is 3, is it right?
　
　
　
Are these parameters for Adam correct?
QAU-Net():
・learning rate
`torch.optim.Adam(lr=0.0001)`

Are these parameters for CrossEntropyLoss correct?
・loss function (cross entropy)
`torch.nn.CrossEntropyLoss(weight=torch.Tensor([0.2, 0.3, 0.5]), reduction='mean')`

・how to call QAU-Net: I think in_ch is a channel, so in_ch is 1 because the LiTS dataset is a gray scale image, is it right?
QAU_Net(in_ch, out_ch) is QAU_Net(1, 3)?

・how to distribute datasets:
1, Choose one case from LiTS dataset cases.
2, Extract 15 images from ct images of the case.
3, Input those images with (15, 1, 512, 512) to QAU_Net(1, 3).
4, Calculate the loss by using outputs (15, 3, 512, 512) and labels (15, 512, 512).
